### ISSUE TITLE
vernemq version command line is empty

### ISSUE THREAD

**Issue by manticorps** [vernemq/vernemq#1119]
### Description
The version command line is not working. 

### Environment

- VerneMQ Version: docker : erlio/docker-vernemq:1.7.1-1-alpine 
- OS: (docker host : archlinux)
- Erlang/OTP version (if building from source): See docker
- VerneMQ configuration (vernemq.conf) : Default 
- Cluster size/standalone: 1

### Expected behavior

Connect to docker and execute vernemq version must show the actual version.
```
docker exec -it verne /bin/sh
$ vernemq version
-> some version text
```
### Actual behaviour
```
docker exec -it verne /bin/sh
$ vernemq version
(empty)
```

### Note 
tested on docker TAG : 1.7.1 / 1.7.0 / 1.6.2

**Comment by ioolkos** [comment]
@manticorps thanks for the reminder, for some reason the command is onlybuilt for the packages but not docker.

**Comment by larshesel** [comment]
Fixed in #1193
